torch/_dynamo/output_graph.py

```diff
@@ -1092,6 +1094,9 @@ def compile_and_call_fx_graph(self, tx, rv, root):
         # free a bit of memory
         self.real_value_cache.clear()
 
+        if config.do_not_emit_runtime_asserts:
+            self.graph.eliminate_dead_code()
```

Comment:
This seems suspicious, because the graph here can have mutations in it, I don't think this is actually safe.